apps/andi/config/test.exs

```diff
@@ -44,6 +44,7 @@ System.put_env("ANDI_SUCCESS_COLOR", "#1170C8")
 System.put_env("ANDI_ERROR_COLOR", "#1170C8")
 System.put_env("ANDI_FOOTER_LEFT_SIDE_TEXT", "Some Left Side Text")
 System.put_env("ANDI_FOOTER_LEFT_SIDE_LINK", "https://www.example.com")
+System.put_env("AUTH0_CLIENT_SECRET", "Example Secret")
```

Comment:
This change was just because I kept seeing "AUTO0_CLIENT_SECRET is required" warnings in tests and it bothered me.